tests/integration/test_vulnerability_detector/test_feeds/debian/test_extra_tags_debian_feed.py

```diff
@@ -0,0 +1,99 @@
+
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import file
+from wazuh_testing.tools.utils import insert_xml_tag
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.services import control_service
+import wazuh_testing.vulnerability_detector as vd
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=2)
+
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, '..', '..', 'data')
+configurations_path = os.path.join(test_data_path, 'configuration', 'test_feeds', vd.INVALID_DEBIAN_FEEDS_CONF)
+custom_debian_oval_feed_path = os.path.join(test_data_path, 'feeds', vd.CUSTOM_DEBIAN_OVAL_FEED)
```

Comment:
You know, same as before.

Reply:
This is something inherited that is being used in all the tests, so we will open issue and fix everything together :)